info.json

```diff
@@ -23,13 +22,29 @@
     "iconLarge": null,
     "fsrMinCompatibility": "7.2.0",
     "date": "2022-02-15T12:40:22+00:00",
-    "exported_from": "tenant.fortisoar.com",
-    "exported_by": "CS Admin",
     "contents": {
         "playbooks": [
             {
                 "name": "09 - ChatOps"
             }
+        ],
+        "globalVariables": [
+            {
+                "id": 7,
+                "name": "Server_fqhn",
+                "value": "master.fortisoar.in",
```

Comment:
Why we need this?

Reply:
@cs-suyog-jinturkar In chatops Bot command result, for example, `$Get Alert <AlertID>` we providing the alert link. In such a case `Server_fqdn` global variable is required.